core/src/commonTest/resources/logback-test.xml

```diff
@@ -0,0 +1 @@
+<configuration/>
```

Comment:
is this ok?

Reply:
@fedefernandez added that, I guess it may be needed to enable logback output on the tests but not sure

Reply:
🤷‍♂️

Reply:
This is for disabling test logging. It can be removed.